### PR TITLE
CH-4426 save touches to touch series tables

### PR DESCRIPTION
**Pull request by yunanzhao**
added dynamic dynamodb feature to rotate on a day of each month

### CODE REVIEW

**Comment by yunanzhao** [comment]
@acquiachrisnagy please take a look at this pull request

File: dynamic_dynamodb/config/config_file_parser.py [hunk 1]
```diff
@@ -263,7 +263,13 @@
     	'option': 'rotate_scavenge',
     	'required': False,
     	'type': 'int'
-    },    
+    },   
+    {
```
Comment (acquiachrisnagy) [inline dynamic_dynamodb/config/config_file_parser.py:267]: Can you change this to be rotate_interval_unit (month)?
Add rotate_window_start to be the minutes in advance before creating the next time series table

Comment (yunanzhao) [inline dynamic_dynamodb/config/config_file_parser.py:267]: rotate_window_start and rotate_interval_unit added. Please review again

File: dynamic_dynamodb/__init__.py [hunk 2]
```diff
@@ -101,6 +104,126 @@ def main():
     except Exception as error:
         logger.exception(error)
 
+def create_time_series_tables(rotate_suffix, rotate_window_start, rotate_scavenge, table_name, table_key, tables_and_gsis, next_table_names):
```
Comment (acquiachrisnagy) [inline dynamic_dynamodb/__init__.py:107]: Can you combine these two methods into one? I think most of the logic is the same except getting the next/prev table name of a period

Comment (yunanzhao) [inline dynamic_dynamodb/__init__.py:107]: There are really only 2 or 3 lines of common functionality shared between the 2 functions. all other lines are different. I dont think merging the 2 makes sense.

**Pull request merged**